libs/designer-ui/src/lib/arrayeditor/plugins/CollapsedArrayValidation.tsx

```diff
@@ -1,24 +1,22 @@
-import type { SimpleArrayItem, ComplexArrayItem } from '..';
+import type { SimpleArrayItem, ComplexArrayItems } from '..';
 import type { ValueSegment } from '../../editor';
-import { ValueSegmentType } from '../../editor';
 import { serializeEditorState } from '../../editor/base/utils/editorToSegement';
-import { showCollapsedValidation, getChildrenNodes, isValidArray } from '../../editor/base/utils/helper';
+import { showCollapsedValidation } from '../../editor/base/utils/helper';
 import { serializeSimpleArray, serializeComplexArray } from '../util/serializecollapsedarray';
+import { getOneDimensionalSchema } from '../util/util';
 import { useLexicalComposerContext } from '@lexical/react/LexicalComposerContext';
 import { OnChangePlugin } from '@lexical/react/LexicalOnChangePlugin';
-import { guid } from '@microsoft-logic-apps/utils';
 import type { EditorState } from 'lexical';
-import { $getRoot } from 'lexical';
 import { useState } from 'react';
 
 export interface CollapsedArrayValidationProps {
   className?: string;
   defaultErrorMessage: string;
   isValid: boolean;
   collapsedValue?: ValueSegment[];
-  itemSchema?: string | string[];
+  itemSchema?: any;
```

Comment:
how would this work as an 'unknown' with some typeguards rather than any?

Reply:
yep we can change this to unknown, don't think itemSchema should be allowed to change